vk/test/routes_test.go

```diff
@@ -32,7 +32,7 @@ func (vts *VektorSuite) SetupTest() {
 	server := vk.New(
 		vk.UseLogger(logger),
 		vk.UseAppName("vk tester"),
-		vk.UseEnvPrefix("APP_"),
+		vk.UseEnvPrefix("APP"),
```

Comment:
Purposely reversed this to `APP` for testing purposes. Do we want two tests / servers to ensure both work?